packages/node_modules/@webex/plugin-meetings/src/meeting/util.js

```diff
@@ -88,8 +88,9 @@ MeetingUtil.cleanOptions = (options, info, owner, userId) => {
     if (MeetingUtil.isOwnerSelf(owner, userId)) {
```

Comment:
I think we should take a look at modifying isOwnerSelf rather than changing the logic here?